core/src/main/java/com/novoda/landingstrip/TabsContainer.java

```diff
@@ -57,18 +57,13 @@ boolean isEmpty() {
 
     void startWatching(final ViewPager viewPager, final ViewPager.OnPageChangeListener onPageChangeListener) {
         if (hasTabs()) {
-            getTabAt(0).getViewTreeObserver().addOnGlobalLayoutListener(
-                    new ViewTreeObserver.OnGlobalLayoutListener() {
-                        @Override
-                        public void onGlobalLayout() {
-                            ViewTreeObserver observer = getTabAt(0).getViewTreeObserver();
-                            removeOnGlobalLayoutListener(observer, this);
-                            viewPager.addOnPageChangeListener(onPageChangeListener);
-
-                            onPageChangeListener.onPageScrolled(viewPager.getCurrentItem(), 0, 0);
-                        }
-                    }
-            );
+            viewPager.addOnPageChangeListener(onPageChangeListener);
```

Comment:
so it used to only add the listener when `onGlobalLayout` was called, now it does it instantly. Is this different?